AirCasting/SessionsSynchronization/Adapters/DataStore/SessionSynchronizationDatabase.swift

```diff
@@ -0,0 +1,152 @@
+        let measurements = measurementStreams?.map { stream -> SessionsSynchronization.SessionStoreMeasurementStreamData in
+            // TODO: Are those force unwraps safe here?
+            return SessionsSynchronization.SessionStoreMeasurementStreamData(id: stream.id!,
+                                                                             measurementShortType: stream.measurementShortType!,
+                                                                             measurementType: stream.measurementType!,
+                                                                             sensorName: stream.sensorName!,
+                                                                             sensorPackageName: stream.sensorPackageName!,
+                                                                             thresholdHigh: Int(stream.thresholdHigh),
+                                                                             thresholdLow: Int(stream.thresholdLow),
+                                                                             thresholdMedium: Int(stream.thresholdMedium),
+                                                                             thresholdVeryHigh: Int(stream.thresholdVeryHigh),
+                                                                             thresholdVeryLow: Int(stream.thresholdVeryLow),
+                                                                             unitName: stream.unitName!,
+                                                                             unitSymbol: stream.unitSymbol!)
+        }
+        return SessionsSynchronization.SessionStoreSessionData(uuid: uuid,
+                                                               contribute: contribute,
+                                                               endTime: endTime,
+                                                               gotDeleted: gotDeleted,
+                                                               isIndoor: isIndoor,
+                                                               name: name!,
+                                                               startTime: startTime!,
+                                                               tags: tags,
+                                                               urlLocation: urlLocation,
+                                                               version: Int(version),
+                                                               longitude: location?.longitude,
+                                                               latitude: location?.latitude,
+                                                               sessionType: type.rawValue,
+                                                               measurementStreams: measurements ?? [])
+    }
+}
+
+extension SessionsSynchronization.Metadata {
+    init(entity: Database.Session) {
```

Comment:
IMo not a good idea. there should be a mapping layer for it to mapping errors spread around the project

Reply:
You're right, I'll make a new class responsible for mapping between `Database <-> SessionSyncStore` objects

Reply:
Edit: no new object needed, `SynchronizationDataConverter` will do.